spi/core-spi/src/test/java/org/eclipse/dataspaceconnector/spi/ObservableTest.java

```diff
@@ -1,5 +1,6 @@
 package org.eclipse.dataspaceconnector.spi;
```

Comment:
this should go into the `observe` package

Reply:
ah missed that one, will do